### PR TITLE
Add ut for parseJSONValue function to cover the JSONPath main parse ability

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Add ut for parseJSONValue function to cover the JSONPath main parse ability.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controllers/applicationfailover/common_test.go [hunk 1]
```diff
@@ -186,6 +187,8 @@ func TestDistinguishUnhealthyClustersWithOthers(t *testing.T) {
 }
 
 func Test_parseJSONValue(t *testing.T) {
+	// This json value describes a DeploymentList object, it contains two deployment element.
```
Comment (RainbowMango) [inline pkg/controllers/applicationfailover/common_test.go:190]:
```suggestion
	// This json value describes a DeploymentList object, it contains two deployment elements.
```

Comment (XiShanYongYe-Chang) [reply pkg/controllers/applicationfailover/common_test.go]: Thank, updated.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**